script/urlChecker.ts

```diff
@@ -38,15 +44,17 @@ async function checkPages(bookSlug: string, pages: string[]) {
   const notFound: string[] = [];
 
   const visitPage = async(page: string) => {
+    const validatedUrl = getPageOrRedirectedUrl(redirects, page);
```

Comment:
this is gonna return _one of_ the redirects targeting the current page in no particular order, i think we need to check all of them